_includes/setup/instructions/geyser/selfhost/viaproxy.md

```diff
@@ -0,0 +1,50 @@
+# ViaProxy Setup
+
+1. [Setup ViaProxy](https://github.com/ViaVersion/ViaProxy?tab=readme-ov-file#usage-for-players-gui).
+2. Download the plugin from [the download page](https://download.geysermc.org/v2/projects/geyser/versions/latest/builds/latest/downloads/viaproxy).
```

Comment:
The download page is `https://geysermc.org/download`

Reply:
We do the same on other platform setup instructions (linking the direct download link)